### DISCUSSION TITLE
请问是否有类继承图的demo呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [didi/LogicFlow#1786] [Q&A]
请问是否有类继承图的demo呢？

类似：这样：
![image](https://github.com/user-attachments/assets/1d13be8a-d977-4f8c-a918-0c0cafe38527)

并且可以给node添加d点击事件。

**Comment by DymoneLewis** [comment]
没有现成直接可以使用的demo，不过这种结构的图用LogicFlow实现起来成本也不高，可以参考自定义节点示例的实现逻辑实现这样的节点https://site.logic-flow.cn/examples/node/custom/#html
关于节点的点击事件，LogicFlow内部在节点被点击的时候会触发node:click事件，可以通过监听方法捕获，事件机制介绍可以看这里：https://site.logic-flow.cn/tutorial/basic/event#%E7%9B%91%E5%90%AC%E4%BA%8B%E4%BB%B6
